.circleci/config.yml

```diff
@@ -46,6 +46,8 @@ jobs:
           name: Run tests - google.cloud.bigtable
           command: |
             if [[ -n $(grep bigtable ~/target_packages) ]]; then
+              test_utils/scripts/circleci/prepare_bigtable.sh
+              export DOWNLOAD_BIGTABLE_SERVER=0 
```

Comment:
Just so that people don't panic or get too excited that we're downloading the *real* Bigtable server, this should really be `DOWNLOAD_BIGTABLE_TEST_SERVER`. :-)